DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: worldfootballR
 Title: Extract and Clean World Football (Soccer) Data
-Version: 0.6.5.0008
+Version: 0.6.6.0000
```

Comment:
Since `fb_league_stats(..., team_or_player = "player")` seems to be a popular function that was buggy before, I think this deserves a little more than a dev version increment!

Reply:
Absolutely, good choice.